src/sparseml/pytorch/torch_to_onnx_exporter.py

```diff
@@ -147,9 +147,10 @@ def post_validate(self, model: onnx.ModelProto) -> onnx.ModelProto:
             raise TypeError(f"Expected onnx.ModelProto, found {type(model)}")
         return model
 
-    def transform(self, module: torch.nn.Module) -> onnx.ModelProto:
-        tmp = tempfile.NamedTemporaryFile("w")
-        file_path = tmp.name
+    def transform(
+        self, module: torch.nn.Module, file_name: str = "model.onnx"
```

Comment:
let's make file_name required here instead since it will always be invoked by `export` which has a file name